PF/Assignment2/PascalTringle.java

```diff
@@ -0,0 +1,62 @@
+public class PascalTringle
+{
+
+	public static String spaces(int row, int n)
+	{
+		String spaceOutput = "";
+		int space;
+		if (row > n) {
+			space = row - n;
+		} else {
+			space = n - row;
+		}
+
+		for (int count = 0; count < space; count++) {
+
+			spaceOutput = spaceOutput.concat(" ");
```

Comment:
ACTION: It is always better to use 'StringBuilder' as you can easily append texts, moreover it boosts performance.